site/docs/webrtc/overview.md

```diff
@@ -21,13 +21,13 @@ Bandwidth’s signaling implementation relies on session and participant objects
 
 ## Streams
 
-For any camera or microphone that you want to connect to Bandwidth WebRTC you will be working with the [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream) that it creates. MediaStreams must be tied to a DOM element on a webpage and a Participant can subscribe to more than one stream, as is the case for audio and video streams, or even multiple video streams as is often the case when screen sharing. Streams can also be aliased so that Participants can subscribe to a particular subset of Streams.
+For any camera or microphone that you want to connect to Bandwidth WebRTC you will be working with the [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream) that it creates. MediaStreams must be tied to a DOM element on a webpage and a participant can subscribe to more than one stream, as is the case for audio and video streams, or even multiple video streams as is often the case when screen sharing. Streams can also be aliased so that participants can subscribe to a particular subset of Streams.
```

Comment:
I think we should end the sentence "MediaStreams must be tied to a DOM element on a webpage.", removing the "and" and starting a new sentence with "A participant....".

This is unrelated to these changes but I think the bold on "Signaling" in the first paragraph of the document is unnecessary and out of place.